### PR TITLE
Draft: Cloud Native Security White Paper Chinese Edition 2

### PR DESCRIPTION
**Pull request by rootsongjc**
Ref: https://github.com/cncf/tag-security/issues/1055

### CODE REVIEW

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 1]
```diff
@@ -0,0 +1,888 @@
...
+  - [XenonStack](https://www.xenonstack.com/) (alok@xenonstack.com), @nyrahul(Accuknox), @ranio1, @lizrice,
+  @justincormack
+
+<!-- cspell:enable -->
+
+## 执行摘要
+
+### 宗旨
+
+技术行业已经转向被看作“云原生”的开发和部署模式。与此同时，技术、产品、标准和解决方案的生态系统正在扩展，挑战决策者保持对复杂设计的了解。特别是 CISO 的角色，需要在这个动态的领域中阐明业务价值主张。与此同时，云原生模式也鼓励了集成安全实践的现代工作流程的消费和采用。
```
Comment (Michelle951) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:36]:
```suggestion
技术行业已经转向被看作“云原生”的开发和部署模式。与此同时，技术、产品、标准和解决方案的生态系统正在扩展，促使决策者及时了解最新的复杂设计。特别是 CISO 的角色，需要在这个动态的领域中阐明业务价值主张。与此同时，云原生模式也推动了消费模型的转变，促进了现代工作流程的应用。现代工作流程要求集成安全实践，例如敏捷方法论和 DevOps 流程。
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 2]
```diff
@@ -0,0 +1,888 @@
...
+
+#### 问题分析
+
+由于明确关注快速开发和部署，因此该领域内的安全问题很复杂。此外，在传统的基于边界的安全模型中，依赖于网络 IP 地址等静态标识符是不切实际的。这种复杂性需要范式转变，通过迁移从纯边界为基础的方法保护应用程序，到基于属性和元数据（如标签和标记）的方法。这种方法可以识别和保护负载，以与云原生应用程序的规模相一致，同时适应不断的变化。这些范式转变需要在应用程序生命周期中增加安全控制的自动化和安全设计架构（例如，零信任）。安全实现的权衡仍然涉及组织内的多个利益相关者，并且显著影响开发人员和运营人员追求业务目标的生产力。云原生应用程序仍然需要开发、分发、部署和运行时操作，但是这种范式指定了新的安全机制，通过这些机制可以高效地实  现这些目标。云原生开发可以建模为构成应用程序生命周期的不同阶段：“开发”、“分发”、“部署”和“运行时”。“云原生安全”与传统的安全方法不同，因为有一个巨大的机会，可以确保安全贯穿于这些不同阶段，而不是通过独立的安全管理干预来结束生命周期。这些概念、工具和过程的持续学习对长期采用和应用至关重要。
```
Comment (Michelle951) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:40]:
```suggestion
由于明确关注快速开发和部署，因此该领域内的安全问题很复杂。此外，在传统的基于边界的安全模型中，依赖于网络 IP 地址等静态标识符是不切实际的。这种复杂性要求我们进行范式转变以保护应用程序安全，放弃基于边界的传统方式，让安全和工作负载（基于标签和标记等属性和元数据进行标识）的联系更加紧密。这种方法可以识别和保护负载，满足云原生应用程序对规模的要求，同时适应不断的变化。这些范式转变需要在应用程序生命周期中增加安全控制的自动化和安全设计架构（例如，零信任）。安全实现的权衡仍然涉及组织内的多个利益相关者，并且显著影响开发人员和运营人员追求业务目标的生产力。云原生应用程序仍然需要开发、分发、部署和运行时操作，但是这种范式指定了新的安全机制，通过这些机制可以高效地实  现这些目标。云原生开发可以建模为构成应用程序生命周期的不同阶段：“开发”、“分发”、“部署”和“运行时”。“云原生安全”与传统的安全方法不同，因为有一个巨大的机会，可以确保安全贯穿于这些不同阶段，而不是通过独立的安全管理干预来结束生命周期。这些概念、工具和过程的持续学习对长期采用和应用至关重要。
```

Comment (mengjiao-liu) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:40]:
```suggestion
由于明确关注快速开发和部署，因此该领域内的安全问题很复杂。此外，在传统的基于边界的安全模型中，依赖于网络 IP 地址等静态标识符是不切实际的。这种复杂性需要范式转变，通过迁移从纯边界为基础的方法保护应用程序，到基于属性和元数据（如标签和标记）的方法。这种方法可以识别和保护负载，以与云原生应用程序的规模相一致，同时适应不断的变化。这些范式转变需要在应用程序生命周期中增加安全控制的自动化和安全设计架构（例如，零信任）。安全实现的权衡仍然涉及组织内的多个利益相关者，并且显著影响开发人员和运营人员追求业务目标的生产力。云原生应用程序仍然需要开发、分发、部署和运行时操作，但是这种范式指定了新的安全机制，通过这些机制可以高效地实 现这些目标。云原生开发可以建模为构成应用程序生命周期的不同阶段：“开发”、“分发”、“部署”和“运行时”。“云原生安全”与传统的安全方法不同，因为有一个巨大的机会，可以确保安全贯穿于这些不同阶段，而不是通过独立的安全管理干预来结束生命周期。这些概念、工具和过程的持续学习对长期采用和应用至关重要。
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 3]
```diff
@@ -0,0 +1,888 @@
...
+
+软件供应链安全在支持更快软件迭代的模型中尤为重要。云原生应用程序生命周期需要包括验证工作负载本身的完整性以及工作负载创建和操作方式的方法。这个挑战被必要的、实际的和一致的使用开源软件和第三方运行时镜像（包括上游依赖的层）所放大。生命周期管道中存在的工件（例如容器镜像）需要持续自动扫描和更新，以确保免受漏洞、恶意软件、不安全的编码实践和其他不良行为的影响。完成这些检查后，将工件进行加密签名以确保完整性并强制执行不可否认性非常重要。值得注意的是，不可变的镜像二进制和镜像的不可变 URL 也非常适合于安全分发。
+
+#### 部署
+
+在整个开发和集成发布阶段，应对候选工作负载的安全性进行实时和持续的验证，如，对签名的工件进行校验，确保容器镜像安全和运行时安全，并可验证主机的适用性。安全工作负载的监控能力，应以可信的方式监控日志和可用指标，与工作负载一同部署来完善整体的安全性。
+
+#### 运行时
+
+云原生运行环境本身可以分解为一系列相互关联的具有独特安全问题[^1]的组件层次结构，例如硬件、主机、操作系统、网络、存储、容器镜像运行时和编排。容器运行时由不同的实现组成，用于各种隔离边界级别，例如共享内核、微型虚拟机沙盒和可信执行环境沙盒。选择满足预期安全要求的运行时非常关键。例如，对于在多租户环境中运行的不受信任的工作负载，可以考虑基于虚拟机的沙盒。对于隐私敏感的金融数据处理，像机密容器这样的可信执行环境（每个进程或虚拟机的内存加密硬件）可能值得考虑。在典型的云原生运行环境中，应用程序通常由几个独立的、单一目的的微服务组成，它们通过服务层抽象相互通信，容器编排层使其成为可能。保护这种相互关联的组件体系结构的最佳实践包括确保只有经过授权的进程在容器命名空间内运行，防止和警报未经授权的资源访问尝试，以及监控网络流量以检测敌对入侵活动。服务网格是另一种抽象，提供了合并和补充有序服务的功能，而无需对工作负载软件本身进行更改（例如，API 流量记录、传输加密、可观察性标记、身份验证和授权）。
```
Comment (mengjiao-liu) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:58]:
```suggestion
云原生运行环境本身可以分解为一系列相互关联的具有独特安全问题[^1]的组件层次结构，例如硬件、主机、操作系统、网络、存储、容器镜像运行时和编排。容器运行时由不同的实现组成，用于各种隔离边界级别，例如共享内核、微型虚拟机沙盒和可信执行环境沙盒。选择满足预期安全要求的运行时非常关键。例如，对于在多租户环境中运行的不受信任的工作负载，可以考虑基于虚拟机的沙盒。对于隐私敏感的金融数据处理，像机密容器这样的可信执行环境（每个进程或虚拟机的内存加密硬件）可能值得考虑。在典型的云原生运行环境中，应用程序通常由几个独立的且单一用途的微服务组成，它们通过服务层抽象相互通信，容器编排层使其成为可能。保护这种相互关联的组件体系结构的最佳实践包括确保只有经过授权的进程在容器命名空间内运行，防止和警报未经授权的资源访问尝试，以及监控网络流量以检测敌对入侵活动。服务网格是另一种抽象，提供了合并和补充有序服务的功能，而无需对工作负载软件本身进行更改（例如，API 流量记录、传输加密、可观察性标记、身份验证和授权）。
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 4]
```diff
@@ -0,0 +1,888 @@
...
+
+### 推荐做法
+
+云原生安全旨在确保与传统安全模型相同或更高的勤勉、诚信、信任和威胁预防条件，同时融合了瞬息万变、分布式和不可变性的现代概念。在这些快速变化的环境中，为了确保安全结果，需要与开发流水线一致的自动化。组织应快速采用这些云原生概念，以帮助在其云原生旅程中创造价值驱动的安全结果。通过尽早将安全整合到开发生命周期中，甚至是通过互动式开发者培训更早地进行整合，安全组织可以实现预防性安全，而不是反应性安全（也可参见“[9 Box of Controls](https://blogs.blackberry.com/en/2016/03/the-9-box-of-controls)”）。强烈建议组织根据相关攻击框架[^2]对安全堆栈进行评估，以明确防御堆栈涵盖哪些威胁。此外，组织需要采用能够左移安全[^3]的方法和方法论，实现 DevOps，并具有足够的灵活性以适应未来的技术进步。
```
Comment (mengjiao-liu) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:62]:
```suggestion
云原生安全旨在确保与传统安全模型相同或更高的勤勉、诚信、信任和威胁预防条件，同时融合了瞬息万变、分布式和不可变性的现代概念。在这些快速变化的环境中，为了确保安全结果，需要与开发流水线保持一致的自动化。组织应快速采用这些云原生概念，以帮助在其云原生旅程中创造价值驱动的安全结果。通过尽早将安全整合到开发生命周期中，甚至是通过互动式开发者培训更早地进行整合，安全组织可以实现预防性安全，而不是反应性安全（也可参见“[9 Box of Controls](https://blogs.blackberry.com/en/2016/03/the-9-box-of-controls)”）。强烈建议组织根据相关攻击框架[^2]对安全堆栈进行评估，以明确防御堆栈涵盖哪些威胁。此外，组织需要采用能够左移安全[^3]的方法和方法论，实现 DevOps，并具有足够的灵活性以适应未来的技术进步。
```

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 5]
```diff
@@ -0,0 +1,888 @@
...
+
+云原生架构是一组模式和设计，为组织提供了一种有效、可持续的无缝抽象，可跨不同的云实例工作，使应用程序栈成为云无关的。容器和微服务架构的采用和创新带来了许多挑战。安全领导人的任务是通过采用预防、检测和应对网络威胁的最佳实践来保护人类[^4]和非人类的资产，同时满足严格的合规要求。一个常见的历史叙事是，安全实施妨碍了 DevOps 团队的速度和敏捷性。因此，安全领导必须实现更紧密的集成和双向理解，赋予 DevOps 团队创建共享的网络风险所有权的能力。
+
+组织应该在现代应用程序开发生命周期中采用安全的云原生模式和架构。特别强调安全架构与组织安全目标之间的协同作用，如零信任、安全软件供应链和 DevSecOps，在云基础设施方面应该优先考虑安全。
+
+本文所描述的概念并不旨在偏袒一种服务、组件或产品，而是可以应用于任何服务选择。
+
+本文档不会提供关于安全或云计算的概念方面的一般教育。也不推荐使用特定的技术或工具。但可能会引用一些解决所讨论主题的技术或工具的示例。
+
+除了本文中的建议外，与数据保护和隐私监管法规要求相关的特定数据安全处理实践，如 GDPR、PCI DSS，可能需要额外的监管特定考虑因素。我们建议读者咨询适当的独立资源，以获取有关任何此类技术控制和合规风险事项的指导。
```
Comment (mengjiao-liu) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:86]:
```suggestion
除了本文中的建议外，与数据保护和隐私监管法规要求相关的特定数据安全处理实践，如 GDPR、PCI DSS，可能需要额外的监管特定考虑因素。我们建议读者查询适当的独立资源，以获取有关任何此类技术控制和合规风险事项的指导。
```
”咨询“ 资源不符合中文习惯

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 6]
```diff
@@ -0,0 +1,888 @@
...
+
+云原生安全旨在确保与传统安全模型相同或更高的勤勉、诚信、信任和威胁预防条件，同时融合了瞬息万变、分布式和不可变性的现代概念。在这些快速变化的环境中，为了确保安全结果，需要与开发流水线一致的自动化。组织应快速采用这些云原生概念，以帮助在其云原生旅程中创造价值驱动的安全结果。通过尽早将安全整合到开发生命周期中，甚至是通过互动式开发者培训更早地进行整合，安全组织可以实现预防性安全，而不是反应性安全（也可参见“[9 Box of Controls](https://blogs.blackberry.com/en/2016/03/the-9-box-of-controls)”）。强烈建议组织根据相关攻击框架[^2]对安全堆栈进行评估，以明确防御堆栈涵盖哪些威胁。此外，组织需要采用能够左移安全[^3]的方法和方法论，实现 DevOps，并具有足够的灵活性以适应未来的技术进步。
+
+### 结论
+
+在组织中战略性地实施云原生安全时，可以在规模上为客户和开发人员提供高可用性、保证性、弹性和冗余性，以确保他们以预期的速度安全地访问所需资源。安全本身仍然是一个跨学科的领域，不能被隔离出来，也不能被视为纯粹的技术领域。开发人员、运维人员和安全人员必须合作、交流和协作，以继续推动该领域和行业的发展。与任何技术创新一样，真正推动社区和云原生安全发展的是那些充满激情踏上这一旅程的人们。
+
+## 介绍
+
+本文旨在为组织和其技术领导层提供云原生安全的简明解释，以及帮助他们将其融入生命周期过程并考虑最适用的应用。云原生安全是一个多目标且多限制的问题领域，涵盖了许多专业和实践领域。几乎所有的 Day 1 和 Day 2 的操作都与安全领域有关，从身份管理到存储解决方案。然而，云原生安全涵盖的领域不止于此；它也是一个涉及个人、团队和组织的人类问题领域。它涉及人类和系统与云原生应用和技术进行交互、更改的机制、过程和意图。
```
Comment (windsonsea) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:70]:
```suggestion
本文旨在为组织和其技术领导层提供云原生安全的简明解释，以及帮助他们将其融入生命周期过程并考虑最适用的应用。云原生安全是一个多目标且多限制的问题领域，涵盖了许多专业和实践领域。从身份管理到存储解决方案，几乎所有的 Day 1 和 Day 2 的操作都与安全领域有关。然而，云原生安全涵盖的领域不止于此；它也是一个涉及个人、团队和组织的人类问题领域。它涉及人类和系统与云原生应用和技术进行交互、更改的机制、过程和意图。
```
放前面读上去更顺一些

File: security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md [hunk 7]
```diff
@@ -0,0 +1,888 @@
...
+
+持续集成（CI）服务器应该被隔离并限制在相似安全分类或敏感性的项目中。需要提升权限的基础设施构建应该在单独的专用 CI 服务器上运行。构建策略应该在 CI 管道和编排器的准入控制器中执行。
+
+供应链工具可以收集并签署构建管道元数据。随后阶段可以验证签名以确认先决条件管道阶段已运行。
+
+读者应确保 CI 和持续交付（CD）基础设施尽可能安全。例如，应优先安装安全更新，并通过使用 HSM 或凭证管理器来保护加密密钥免受外泄。
+
+#### 镜像扫描
+
+扫描容器镜像是保护容器应用程序整个生命周期中最常见的安全方式之一。在将镜像部署到生产环境之前，在 CI 流水线中进行扫描非常重要。此外，持续扫描正在运行的容器镜像同样重要，以识别新发现的漏洞。整合此功能可确保开发人员、运营商和安全专家拥有所有已知漏洞和详细信息，例如严重性、[通用漏洞评分系统（CVSS）](https://nvd.nist.gov/vuln-metrics/cvss)评分和可用的缓解/修复措施。将容器镜像的漏洞扫描与流水线合规规则相结合，可确保只部署已充分修补的应用程序，从而减少潜在攻击面。容器镜像扫描还有助于识别开源软件包或从开源镜像库中获取的基础镜像层中的恶意软件存在。虽然容器镜像扫描可以为团队提供漏洞或恶意软件的证据，但它并不能修复漏洞或防止恶意软件。组织需要确保采取行动来解决容器扫描发现，并执行组织合规规则。
```
Comment (windsonsea) [inline security-whitepaper/v2/cloud-native-security-whitepaper-simplified-chinese.md:156]:
```suggestion
扫描容器镜像是保护容器应用程序整个生命周期中最常见的安全方式之一。在将镜像部署到生产环境之前，在 CI 流水线中进行扫描非常重要。
此外，持续扫描正在运行的容器镜像同样重要，以识别新发现的漏洞。整合此功能可确保开发人员、运营商和安全专家拥有所有已知漏洞和详细信息，
例如严重性、[通用漏洞评分系统（CVSS）](https://nvd.nist.gov/vuln-metrics/cvss)评分和可用的缓解/修复措施。将容器镜像的漏洞扫描与流水线合规规则相结合，
可确保只部署已充分修补的应用程序，从而减少潜在攻击面。容器镜像扫描还有助于识别开源软件包或从开源镜像库中获取的基础镜像层中的恶意软件存在。
虽然容器镜像扫描可以为团队提供漏洞或恶意软件的证据，但它并不能修复漏洞或防止恶意软件。组织需要确保采取行动来解决容器扫描发现，并执行组织合规规则。
```
这种很长的一段，是不是可以考虑适当折行，这样 review 看起来会比较方便额

**Review by windsonsea — COMMENTED** [review]
Truly admire the exceptional work that showcases a profound understanding and unwavering enthusiasm for cloud-native technology!

**Review by hwang37 — COMMENTED** [review]
Thanks for the bilingual translation effort!
Do you think it's a good idea to add a new session to list the acronym, name and translation?

I collect some acronym used in the PR:

ABAC:	Attribute-Based Access Control
AES:	Advanced Encryption Standard
AI:	Artificial Intelligence
API: 	 Appliation Programming Interface
BYOK:	Bring Your Own Key
CA:	Certificate Authority
CaaS:	Containers-as-a-Service
CD:	Continuous Delivery
CI:	Continuous Integration
CIS:	Center for Internet Security
CISO:	Chief (Information) Security Officer
CMK:	Customer Master Key
CNCF:	Cloud Native Computing Foundation
CTO: 	Chief Technology Officer 
CVE:	Common Vulnerabilities and Exposures
CVSS:	Common Vulnerability Scoring System
DAAS:	data, assets, applications, and services
DAST:	Dynamic Application Security Testing
DDoS:	Distributed Denial of Service 
DEK:	Data Encryption Key
DoS:	Denial of Service
EBA:	European Banking Authority
EIOPA:	European Insurance and Occupational Pensions Authority
ESMA:	European Securities and Markets Authority
Faas:	Functions-as-a-Service
GID:	Group identifier
HSM:	Hardware Security Modules
IaaS:	Infrastructure as a Service
IaC:	Infrastructure as Code
IAM:	Identity and Access Management
IDE:	Integrated Development Environment
KMS:	Key Management System
MAC:	Mandatory Access Control
ML:	Machine Learning
NFS:	Network File System
OCI:	Open Container Initiative
OPAL:	Opal Storage Specification
OWASP:	Open Web Application Security Project
PaaS:	Platform as a Service
RBAC:	Role-based Access Control
ROI: 	Return On Investment
SAST:	Static Application Security Testing
SBOMs:	Software Bill of Materials
SOC:	System On Chip
SoDï¼š	Separation of duties
SSD:	Solid-State Drive
TEE:	Trusted Execution Environments
TLS:	Transport Layer Security
TPM:	Trusted Platform Module
UID:	Unique Identifier
VEX:	Vulnerability Exploitability eXchange
VM:	 Virtual Machine
VPN:	Virtual Private Network

**Comment by rootsongjc** [comment]
> Thanks for the bilingual translation effort! Do you think it's a good idea to add a new session to list the acronym, name and translation?
> 
> I collect some acronym used in the PR:
> 
> ABAC: Attribute-Based Access Control AES: Advanced Encryption Standard AI: Artificial Intelligence API: Appliation Programming Interface BYOK: Bring Your Own Key CA: Certificate Authority CaaS: Containers-as-a-Service CD: Continuous Delivery CI: Continuous Integration CIS: Center for Internet Security CISO: Chief (Information) Security Officer CMK: Customer Master Key CNCF: Cloud Native Computing Foundation CTO: Chief Technology Officer CVE: Common Vulnerabilities and Exposures CVSS: Common Vulnerability Scoring System DAAS: data, assets, applications, and services DAST: Dynamic Application Security Testing DDoS: Distributed Denial of Service DEK: Data Encryption Key DoS: Denial of Service EBA: European Banking Authority EIOPA: European Insurance and Occupational Pensions Authority ESMA: European Securities and Markets Authority Faas: Functions-as-a-Service GID: Group identifier HSM: Hardware Security Modules IaaS: Infrastructure as a Service IaC: Infrastructure as Code IAM: Identity and Access Management IDE: Integrated Development Environment KMS: Key Management System MAC: Mandatory Access Control ML: Machine Learning NFS: Network File System OCI: Open Container Initiative OPAL: Opal Storage Specification OWASP: Open Web Application Security Project PaaS: Platform as a Service RBAC: Role-based Access Control ROI: Return On Investment SAST: Static Application Security Testing SBOMs: Software Bill of Materials SOC: System On Chip SoD: Separation of duties SSD: Solid-State Drive TEE: Trusted Execution Environments TLS: Transport Layer Security TPM: Trusted Platform Module UID: Unique Identifier VEX: Vulnerability Exploitability eXchange VM: Virtual Machine VPN: Virtual Private Network

Done

**Review by anvega — APPROVED** [review]
Approving based on the reviews of others

**Pull request merged**